cli.py

```diff
@@ -75,14 +75,14 @@ def display_station(station, geocode=False, use_colors=False):
     padding = ' ' * 1
     total_slots = station.get('extra', {}).get('slots')
     if not total_slots:
-        if station['empty_slots']:
+        if station.get('empty_slots') is not None:
             total_slots = station['free_bikes'] + station['empty_slots']
         else:
             total_slots = 0
 
     total_slots = float(total_slots)
     bikes = int(station['free_bikes'])
-    slots = int(station['empty_slots'] or 0)
+    slots = int(station.get('empty_slots') or 0)
```

Comment:
Since this is using `get`, I guess we can use the default argument instead of `or`